litmus/AArch64Compile_litmus.ml

```diff
@@ -1204,10 +1204,25 @@ module Make(V:Constant.S)(C:Config) =
 
     let kernel_mode = [{ empty_ins with memo="svc #471"}]
 
-    let fault_handler_prologue p =
-      map_ins ["b 1f";sprintf  "asm_handler%d:" p]
-
-    and fault_handler_epilogue code =
+    let user_prologue =
+      ["mrs %[tr0],esr_el1";
+       "lsr %[tr0],%[tr0],#26";
+       "cmp %[tr0],#21";
```

Comment:
and this to:
`"cmp %[tr0],#ESR_EL1_EC_SVC64";`

Reply:
Same answer, will try.